### PR TITLE
[MIG][10.0] rating: nothing to do

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

File: addons/rating/migrations/10.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---Fields in module 'rating'---
+rating       / rating.rating            / consumed (boolean)            : NEW
```
Comment (pedrobaeza) [inline addons/rating/migrations/10.0.1.0/openupgrade_analysis_work.txt:2]: This analysis file is not correct... this field exists also in v10, but other new fields like `rating_image` or `rating_text` are not reported. @StefanRijnhart do you have idea why?

Comment (StefanRijnhart) [reply addons/rating/migrations/10.0.1.0/openupgrade_analysis_work.txt:2]: Ah @pedrobaeza you are looking at the analysis of 10.0 so this field should be here. As for rating_image and rating_text in 11.0, these are computed fields and have been traditionally left out of the analysis as they presumably do not need any work.

Comment (pedrobaeza) [reply addons/rating/migrations/10.0.1.0/openupgrade_analysis_work.txt:2]: Agh, I have mixed v10 and v11 as we have both. Sorry for the noise.

Comment (pedrobaeza) [inline addons/rating/migrations/10.0.1.0/openupgrade_analysis_work.txt:2]: @cubells you have to set this to True if `rating != -1`, which is the previous default value when unset.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: addons/rating/migrations/10.0.1.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (pedrobaeza) [reply addons/rating/migrations/10.0.1.0/openupgrade_analysis_work.txt:2]: EDIT: Sorry, the correct comparison value is `!= -1`

Comment (cubells) [reply addons/rating/migrations/10.0.1.0/openupgrade_analysis_work.txt:2]: @pedrobaeza 
done!

**Review by StefanRijnhart — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I'm thinking that for having the same data as in v11, you have to put the field `rating` to 0 when the value is `-1`. Maybe that illegal value makes something to crash.

**Comment by cubells** [comment]
@pedrobaeza 
Can you recheck?

**Comment by pedrobaeza** [comment]
@cubells you have made the contrary. Read again what I wrote. You need 2 queries.

**Comment by cubells** [comment]
Oops. You are right.
thanks @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**